cmd2/utils.py

```diff
@@ -477,11 +477,7 @@ def write(self, s: str) -> None:
         """
         if not isinstance(s, str):
             raise TypeError('write() argument must be str, not {}'.format(type(s)))
-
-        if not self.pause_storage:
-            self.buffer.byte_buf += s.encode(encoding=self.encoding, errors=self.errors)
-        if self.echo:
-            self.inner_stream.write(s)
+        self.buffer.write(s.encode(self.encoding))
```

Comment:
Does dropping `errors=self.errors` have a significant effect?  I would worry about edge cases.

Reply:
Good catch. Added that back.